### PR TITLE
testReadFloatFix

### PR DESCRIPTION
**Pull request by YanOleks**
Added dpTestReadFloat to ConsoleScannderDataProvider
testReadFloat tests output

### CODE REVIEW

**Review by smelovd — CHANGES_REQUESTED** [review]
set param as a first argument in method (check updates on dev)

File: src/test/java/org/academy/codewars/dataproviders/ConsoleScannerDataProvider.java [hunk 1]
```diff
@@ -23,4 +23,20 @@ public Iterator<Object[]> dpTestReadInt() {
...
+
+    @DataProvider(name = "dp-testReadFloat")
+    public Iterator<Object[]> dpTestReadFloat() {
+        List<Object[]> inputs = new ArrayList<>();
+        String param = "test param";
+        String expectedOutput1 = "Enter a " + param + " (float): ";
+        String expectedOutput2 = "Enter a " + param + " (float): " +
+                "Invalid data format. A floating point number is expected.\n" +
+                "Enter a " + param + " (float): ";
+        expectedOutput2 = expectedOutput2.replaceAll("\\n|\\r\\n", System.lineSeparator());
```
Comment (smelovd) [inline src/test/java/org/academy/codewars/dataproviders/ConsoleScannerDataProvider.java:35]: ?, use /n on new line (check completed methods)

**Review by smelovd — CHANGES_REQUESTED** [review]

File: src/test/java/org/academy/codewars/dataproviders/ConsoleScannerDataProvider.java (hunk 1 above)
Comment (YanOleks) [reply src/test/java/org/academy/codewars/dataproviders/ConsoleScannerDataProvider.java:35]: Check screenshot in telegram

**Comment by smelovd** [comment]
resolve conflict

**Review by sbekberov — APPROVED** [review]

**Pull request merged**